modules/base/rendering/renderablesphere.cpp

```diff
@@ -84,6 +84,19 @@ namespace {
         "This value determines percentage of the sphere is visible before starting "
         "fading-out it."
     };
+
+    static const openspace::properties::Property::PropertyInfo FadeInThreshouldInfo = {
+        "FadeInThreshould",
+        "Fade-In Threshould",
+        "Distance from center of MilkyWay from where the astronomical object starts to "
```

Comment:
Some more documentation here?  What units, how does it work, etc

Reply:
Fixed.